### ISSUE TITLE
"nathantreid:css-modules@3.2.1 : No compatible binary build found for this package" error on 'meteor update'

### ISSUE THREAD

**Issue by yched** [nathantreid/meteor-css-modules#123]
When running `meteor update`, I get the following error :
```
This project is already at Meteor 1.8, the latest release.
=> Errors while upgrading packages:           
                                              
While checking for nathantreid:css-modules@3.2.1:
error: No compatible binary build found for this package. Contact the package author and
ask them to publish it for your platform.
```

I then manually updated nathantreid:css-modules to 3.2.0 instead (`meteor add nathantreid:css-modules@3.2.0`), but I still get the same error on `meteor update`, which prevents me from upgrading the other packages.

It seems a 3.2.1 version got somehow published on atmosphere without the associated code ?

**Comment by akanix42** [comment]
I'm not sure how that could happen, but I'll look into it.

**Comment by oswaldoacauan** [comment]
Same error is happening for us. Any news regarding this issue @nathantreid ?

**Comment by met5678** [comment]
Having the same issue here.

**Comment by yched** [comment]
@nathantreid Maybe just issue a new release to get past the buggy thing about 3.2.1 ?

**Comment by akanix42** [comment]
I've published 2 releases, `nathantreid:css-modules@3.2.2-beta.1` using Meteor 1.7 and `nathantreid:css-modules@3.2.2-beta.2` using Meteor 1.8. Please let me know if they work for you - I'm trying to determine if the update to Meteor 1.8 is a breaking change for this package.

**Comment by yched** [comment]
@nathantreid Thanks !

So, testing with our app (Meteor 1.8):
- `meteor update` still blocks on the same error than in the OP, but I guess that it's because your new releases are beta and therefore not official yet.
- Manually installing 3.2.2-beta.2 (`meteor add nathantreid:css-modules@3.2.2-beta.2`) works fine, and then `meteor update` finally works normally and does update the other packages (we were behind on a couple core packages)
- But then running the app (`meteor run`) outputs the error I described in #124 during build. The build does finish (`App running at: http://localhost:3000/`), but the app crashes in the client with "Uncaught Error: Cannot find module './Layout.styles.scss'  in modules-runtime.js:615" - that's the first import of a local stylesheet in the app's import dependency tree.

So it seems republishing new versions fixed `meteor update` (and will fix it for good when the releases are official, for now it still requires manually updating to the betas to skip 3.2.1).
But then something seems broken during compilation with the new versions - and that's what #124 is about.

I should add that our Meteor 1.8 app runs fine with nathantreid:css-modules@3.1.4 so far :-)

**Comment by akanix42** [comment]
Thanks for the feedback - I've published 3.2.2 to resolve the issue.
